src/absil/ilread.fsi

```diff
@@ -24,84 +24,86 @@
 ///     class.  That is not particularly satisfactory, and it may be
 ///     a good idea to build a small library which extracts the information
 ///     you need.  
-module FSharp.Compiler.AbstractIL.ILBinaryReader 
```

Comment:
I think the changes in this file could be minimized by not changing `module` to `namespace` at the top.

Reply:
Yea, I see what I need to do in order to preserve it.